easybuild/easyconfigs/a/ABINIT/ABINIT-7.6.2_odamix.patch

```diff
@@ -0,0 +1,11 @@
+--- src/67_common/odamix.F90.orig	2016-08-22 13:27:30.289621140 +0100
```

Comment:
@AFCJamie same here, please include information about patch & where it comes from